src/server/database/mongodb-adaptor.js

```diff
@@ -8,7 +8,9 @@ class MongoCollectionAdaptor {
   }
 
   put (doc) {
-    return this.collection.insertOne(doc)
+    return new Promise((resolve, reject) => {
+      this.collection.insertOne(doc).then(() => resolve(doc)).catch((err) => reject(err))
+    })
```

Comment:
It doesn't return the document itself, but some status. I'd rather return the document instead in order to be able to chain requests.